src/SmartComponents/BaselinesTable/BaselinesTable.js

```diff
@@ -135,30 +85,52 @@ class BaselinesTable extends Component {
         return rows;
     }
 
-    fetchBaseline = (baselineId) => {
-        const { history } = this.props;
+    renderRadioButton = (data) => {
+        const { onSelect } = this.props;
 
-        history.push('baselines/' + baselineId);
+        return (
+            <React.Fragment>
+                <Radio
+                    aria-label={ 'radio' + data[1] }
+                    isChecked={ data.selected }
+                    onChange={ onSelect }
+                    name={ data[1] }
+                    id={ data[0] }
+                />
+            </React.Fragment>
+        );
     }
 
     renderRows() {
-        const { baselineTableData, kebab } = this.props;
+        const { baselineTableData, kebab, onClick, tableType } = this.props;
         let table = [];
 
         baselineTableData.forEach((baseline) => {
             let row = [];
-            let link = <div>
-                <a className="pointer active-blue"
-                    onClick={ () => this.fetchBaseline(baseline[0]) }
-                >
-                    { baseline[1] }
-                </a>
-            </div>;
-            row.push(link);
+
+            if (tableType === 'RADIO') {
```

Comment:
Should this be based on the type of select that's been configured for the component? `hasSingleSelect` ?